chunky/src/java/se/llbit/chunky/model/DecoratedPotModel.java

```diff
@@ -137,7 +165,7 @@ public DecoratedPotModel(String facing, String[] sherds) {
       // sherds[2] right crafting slot -> east
       getTextureForsherd(sherds[2]),
       Texture.decoratedPotBase, // top
-      Texture.decoratedPotBase  // bottom
+      Texture.decoratedPotSide  // bottom
```

Comment:
is the "// bottom" comment still correct?

Reply:
Yes, for some reason Mojang is sampling a seemingly arbitrary pixel from the side texture to determine the bottom:
![image](https://github.com/chunky-dev/chunky/assets/46458276/4217c23d-2d67-41c9-bf66-5e8bfaa3376d)

Reply:
wait... I can't reproduce this behavior in a fresh unmodded instance, wtf is going on

Reply:
sodium bug (!?)